controllers/apply-next/lib/validate-model.js

```diff
@@ -14,6 +14,7 @@ module.exports = function validateModel(formModel) {
                 'checkbox',
                 'currency',
                 'date',
+                'dateRange',
```

Comment:
Could we either make this hyphenated or make the others camelCase, this is making my eye twitch 🙂

Reply:
Oops yeah good call. Will hyphenate it! Got confused as the Joi object is camel cased.